index.js

```diff
@@ -1,34 +1,46 @@
-var ispnum = function(num) {
-    for(var i = 2; i < num; i++)
-      if  (num % i === 0) return false;
-    return num > 1;
-};
-
-const fibonacci = (num) => {
-    if (num <= 1)return 1;
-  return fibonacci(num - 1) + fibonacci(num - 2);
-};
-
-function nxtPrmFib(number) {
-    let r = 0;
-    let l = 1;
-    while (true) {
```

Comment:
Replace `while/true` with `for(from/to)` cycle. It makes code more predictable and testable.

P.S. Updated code contains an example of code testing.